lib/generators/administrate/install/install_generator.rb

```diff
@@ -40,6 +40,11 @@ def warn_about_invalid_models
           puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
           puts "         It is not connected to a database table."
         end
+
+        unnamed_constants.each do |invalid_model|
+          puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
+          puts "         It is not a valid Ruby constant."
```

Comment:
Is there something that could be linked to the user to help them debug/fix this?